moto/cloudtrail/models.py

```diff
@@ -300,6 +300,14 @@ def get_trail(self, name_or_arn: str) -> Trail:
     def get_trail_status(self, name: str) -> TrailStatus:
         if len(name) < 3:
             raise TrailNameTooShort(actual_length=len(name))
+        current_account = cloudtrail_backends[self.account_id]
+
+        # We need to check if there is a MultiRegion trail created for the region checked
+        for backend in current_account.values():
+            for trail in backend.trails.values():
+                if trail.is_multi_region or trail.region_name == self.region_name:
+                    self.trails[trail.trail_name] = trail
+
```

Comment:
```suggestion
        all_trails = self.describe_trail(include_shadow_trails=True)

        # We have to refactor the next line a little bit, to find the correct trail based on the given name
        trail = next(... for trail in all_trails ... )
```

This way we can reuse the multi-region logic that already exists in `describe..`, and we don't have to deal with duplicate trails

Reply:
Good point though, I'll change it.